Makefile

```diff
@@ -189,6 +190,16 @@ registry-load-bundles: ## Load selected e2e testdata container images created in
 	testdata/bundles/registry-v1/build-push-e2e-bundle.sh ${E2E_REGISTRY_NAMESPACE} $(REGISTRY_ROOT)/bundles/registry-v1/prometheus-operator:v1.2.0 prometheus-operator.v1.2.0 prometheus-operator.v1.0.0
 	testdata/bundles/registry-v1/build-push-e2e-bundle.sh ${E2E_REGISTRY_NAMESPACE} $(REGISTRY_ROOT)/bundles/registry-v1/prometheus-operator:v2.0.0 prometheus-operator.v2.0.0 prometheus-operator.v1.0.0
 
+.PHONY: bootstrap-local-dev
+bootstrap-local-dev: $(KIND) $(SETUP_ENVTEST) #HELP Copies the kubernetes service certificates and starts kubefwd
+	@$(KIND) export kubeconfig --name $(KIND_CLUSTER_NAME) > /dev/null 2>&1
+	@echo "Scaling down operator-controller-controller-manager deployment"
+	@eval $$($(SETUP_ENVTEST) use -p env $(ENVTEST_VERSION) $(SETUP_ENVTEST_BIN_DIR_OVERRIDE)) && kubectl scale deployment -n $(OPERATOR_CONTROLLER_NAMESPACE) operator-controller-controller-manager --replicas=0
+	@echo "Copying operator-controller system namespace service certificates from kind cluster"
+	@echo "set --ca-certs-dir=$(shell hack/tools/copy-service-certs.sh)"
+	@echo "Starting kubefwd - sudo is needed for this"
+	@sudo -E $(KUBEFWD) svc -n $(OPERATOR_CONTROLLER_NAMESPACE)
```

Comment:
Would it make sense for this to be in a separate script (e.g. in `hack/`)?

Reply:
yeah - I'm going to move it there. But, I'll drop the make target for now. Having said that, I generally prefer to put the logic directly in the Makefile. Long is ok, complex is where I'd go to a script.

Reply:
though I'm also amenable to every/most targets are scripts. I just don't want to keep jumping b/w Makefile and script unless there's a good reason